config/settings/test.py

```diff
@@ -13,7 +13,7 @@
 ENV_HOSTS = [host for host in env.str("ALLOWED_HOSTS", "").split(",") if host]
 ALLOWED_HOSTS = ENV_HOSTS + ["localhost", ".localhost", "127.0.0.1", "0.0.0.0"]
 
-GA_TAG_KEY = "TEST_GA_TAG_KEY"
+# GA_TAG_KEYS = env.list("GOOGLE_GA_TAG_KEYS", default=["TEST_GA_TAG_KEY"])
```

Comment:
Is there a reason that this is commented out?

Reply:
Yes. We have it defined in the `base.py` so it's not needed here anymore. I've updated the tests to use the default value . I'll remove the comment